### ISSUE TITLE
[❌ ] Failed to install Snapchat.ipa to iOS device. Check tweakpatcher.log for more information.

### ISSUE THREAD

**Issue by nullpixel** [Defying/TweakPatcher#3]
I followed all the instructions etc, restarted, and updated Xcode, but the application always fails to install on my SE, iOS 9.3.1.

Log attached below.

[tweakpatcher.log.txt](https://github.com/Defying/TweakPatcher/files/211181/tweakpatcher.log.txt)

I'd really appreciate help, as you seem to have missed my post on reddit :D, I don't want to seem pushy, but I don't understand why I get the could not inspect application package in the logs

**Comment by ItzLevvie** [comment]
Have you seen EverythingApplePro's video on how to install this?

**Comment by nullpixel** [comment]
@ItzLevvie Yes, I don't really need guides tbh, I used the github tutorial, and followed it exactly, and I get a iOS-Deploy error, so I know that I have done everything right, I just don't think iOS deploy is liking my SE

**Comment by Starbix** [comment]
Make sure Snapchat is not installed, I think I got this error when installing it while Snapchat was still installed

**Comment by nullpixel** [comment]
Snapchat isn't installed @Pukinator

**Comment by nerdtron123** [comment]
make sure the deploy version in xCode is lower then/equal to your device version.

**Comment by nullr1ver** [comment]
Could be this, lemme check @nerdtron123

**Comment by nullpixel** [comment]
No luck @nerdtron123

**Comment by Defying** [comment]
Do you get the same error when you try to sideload the patched IPA in Xcode?

**Comment by nullpixel** [comment]
Yes, the exact error (Makes me think it's a Xcode issue, but I'm on the latest version) @Defying

**Comment by Defying** [comment]
are you able to resign the original Snapchat IPA with [iOS App Signer](https://dantheman827.github.io/ios-app-signer/) and sideload it with Xcode?

**Comment by ghost** [comment]
I get the same error. Using an iPhone 6 Plus

**Comment by DawsonSeibold** [comment]
I am also using an IPhone 6 Plus and got the error. I tried the IOS App Signer and sideload it to Xcode, Xcode gave an error: `does not contain a valid Info.plist`.

**Comment by nullpixel** [comment]
Yes I am @Defying

**Comment by ghost** [comment]
It didn't work with iMac xCode 7.3 and iPhone 6 Plus on iOS 9.3.1 and iOS 9.3.2. I also tired it on my MacBook Air xCode 7.1 and the iPhone 6 Plus on iOS 9.3.1 always the same Error.
On the MacBook it also gave me this line right before the Error: "./tweakpatcher.sh: line 227:  5573 Abort trap: 6           $IOSDEPLOY -W -b $TWEAK-patched.ipa >> $LOGFILE 2>&1"

Sideloading Snapchat with xCode gave me the Error: "does not contain a valid Info.plist".

[tweakpatcher.log.txt](https://github.com/Defying/TweakPatcher/files/211671/tweakpatcher.log.txt)

**Comment by nullpixel** [comment]
Please can you make your own issue, that looks like a completely unrelated error to mine :D @MpDesigns

**Comment by ghost** [comment]
@nullr1ver Its the same Error

**Comment by nullpixel** [comment]
@MpDesigns With a different reason: You get does not contain a valid Info.plist in your logs `(Assertion failed: (AMDeviceIsPaired(device)), function handle_device, file /Users/marcel/Desktop/TweakPatcher-master/.temp/ios-deploy-1.8.5/src/ios-deploy/ios-deploy.m, line 1618.,` I get `Can't inspect application package`

**Comment by Jwiggiff** [comment]
I'm not sure if this will work but mine wasn't working and I did this and it worked.

Follow the exact same instructions but when you are Creating the Xcode project before you click fix issue make sure the bar at the top loads completely

**Comment by nullpixel** [comment]
Scrap that, @Defying I cannot install the old IPA, even when cracked and from multiple sources. I get the does not contain a valid info.plist

**Comment by nullpixel** [comment]
I did that @Jwiggiff

**Comment by nullpixel** [comment]
WHOOOOOOOO I fixed it!

@MpDesigns To fix your issue do the following steps:

1) Rename yourapp_resigned.ipa to yourapp_resigned.zip
2) Unzip yourapp_resigned.zip to yourapp_resigned folder
3) Launch Terminal, type command: $ cd yourapp_resigned folder
4) Type command: $ chmod +x Payload/yourapp (here yourapp is executable file)
5) Type command $ zip -r -T -y yourapp_resigned_fixed.ipa Payload
6) Try to install yourapp_resigned_fixed.ipa to your phone.

Then for my issue, all I had to do was the above, and I had to delete the _codesigned folder inside the .app. Then I got it all working :D
